lib/scavenger_hunt/app/views/scavenger_hunt/leaderboard/index.html.slim

```diff
@@ -10,12 +10,16 @@ table
       th Stats
   tbody
     - @players.each_with_index do |player, index|
-      tr
-        td= index.succ
-        td= player.name
-        td= player.location_count
-        td
-          = player.aggregate_time
-          br
-          = player.clue_count
-
+      - if player.name.present?
```

Comment:
I changed this because Anna requested that if there is no username, she would like that hidden. Issue #223